sass/_utils.scss

```diff
@@ -2,8 +2,10 @@
     display: none;
 }
 
-a:focus {
-    outline: none;
+// Visual outline on focused elements, for accessibility
+*:focus {
+    border-radius: 5px;
+    outline: $focus-outline;
```

Comment:
This is all quite good, but using the `:focus` pseudo-class results in the focus styling being applied whenever the mouse is held down, in addition to when the keyboard focuses an element. This is likely unintentional, and looks a bit odd on most elements (although I kinda like it on the cards).

I'd recommend using `:focus-visible` instead, which is applied only when the keyboard focuses an element (but ignores focus events from other devices, mostly). Further reading [here](https://css-tricks.com/the-focus-visible-trick/) and [here](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible).

Reply:
Neat, I didn't know about this!